tests/analysis/losses/test_losses.py

```diff
@@ -161,13 +176,14 @@ def create_linestring(row):
             graph_file_hazard=_config.graph_files.base_graph_hazard,
         )
 
-        _losses = Losses(_analysis_input, _config)
+        _losses = SingleLinkLosses(_analysis_input, _config)
```

Comment:
same as earlier either do it for both or use a 'dummy' inherited class

Reply:
Split in 2 modules, 1 for base and 1 for the child losses analyses